### DISCUSSION TITLE
An error on a simple component route

### DISCUSSION THREAD

**Discussion by alecpl** [livewire/livewire#6988] [1. Help] [answered]
Laravel: 10.26.2
Livewire: 3.0.5
Jetstream: 4.0.3
PHP: 8.1.23

I'm doing a simple full page component as documented at https://livewire.laravel.com/docs/components:
```
use App\Livewire\Entity;
Route::get('/entity/domain/{domain}', Entity\Domain::class);
```
However I get an exception:
```
Controller class App\Http\Controllers\App\Livewire\Entity\Domain for one of your routes was not found. Are you sure this controller exists and is imported correctly?
```
What am I doing wrong?

**Comment by joshhanley** [comment, marked as answer]
@alecpl since Laravel 8 the controller routes are no longer prefixed. So you will need to open your `RouteServiceProvider` and remove the `App\Http\Controllers` prefix then hopefully all should work. See here for details https://laravel.com/docs/8.x/upgrade#automatic-controller-namespace-prefixing

Hope this helps!

**Comment by alecpl** [comment]
I felt it's some leftover. Thanks. Fixed.

**Reply by joshhanley** [reply]
@alecpl no worries! 🙂
